### ISSUE TITLE
Shadowmeld should not break upon casts starts until 1.10

### ISSUE THREAD

**Issue by Slicy** [LightsHope/issues#404]
Currently shadowmeld breaks upon starting a cast in all stages of the game. 

It should only break **at the end of casts**.

The racial worked as such for the very large majority of retail vanilla. It changed in 1.10.

- "Stealth and Invisibility effects will now be canceled at the beginning of an action (spellcast, ability use etc...), rather than at the completion of the action." patch 1.10
- http://wow.allakhazam.com/forum.html?forum=21&mid=114159373135259369
- And perhaps 90% of retail vanilla videos starring NEs.

**Comment by Seberats** [comment]
if i remember, all spells functions are on 1.12 patch. that's why.
Btw, i would love a change like that for shadowmeld...

**Comment by gotmilk0112** [comment]
Yeah, when it comes to spells/abilities and talents, everything is 1.12
